src/java/org/apache/cassandra/transport/Server.java

```diff
@@ -454,51 +455,32 @@ public static LatestEvent forTopologyChange(Event.TopologyChange.Change change,
 
         // We keep track of the latest status change events we have sent to avoid sending duplicates
         // since StorageService may send duplicate notifications (CASSANDRA-7816, CASSANDRA-8236, CASSANDRA-9156)
-        private final Map<InetAddress, LatestEvent> latestEvents = new ConcurrentHashMap<>();
+        private final Map<InetAddressAndPort, LatestEvent> latestEvents = new ConcurrentHashMap<>();
         // We also want to delay delivering a NEW_NODE notification until the new node has set its RPC ready
         // state. This tracks the endpoints which have joined, but not yet signalled they're ready for clients
-        private final Set<InetAddress> endpointsPendingJoinedNotification = ConcurrentHashMap.newKeySet();
-
-
-        private static final InetAddress bindAll;
```

Comment:
I looked up the JIRA and it's not necessary anymore. The comment says 

                // Note that after all nodes are running a version that includes CASSANDRA-5899, rpcAddress should
                // never be 0.0.0.0, so this can eventually be removed.

So we can never hit that path anyway. I think it was useful when introduced, but it's not anymore.

Reply:
hmm, looks like I didn't read that comment. But yes, you are correct, so +1 here